### DISCUSSION TITLE
✅ Visibility conditions

### DISCUSSION THREAD

**Discussion by ghost** [elementor/elementor#22341] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

In cases of more complicated sites, it would be very convenient to be able to show content when certain conditions occur. jetEngine allows you to display items based on multiple conditions. I find this really useful.

### Describe the solution you'd like

I would therefore like a new section in Elementor, where to manage the display conditions of each single element.

### Describe alternatives you've considered

Currently the best plugin to do this is undoubtedly [JetEngine](https://crocoblock.com/plugins/jetengine/). But given the significant development made by Elementor in the last updates, I believe that the implementation of this feature can raise the quality of Elementor considerably.

**Comment by nicholaszein** [comment]
Hello there @EnthonySclafani!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.19.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
